draft-eggert-bcp45bis.md

```diff
@@ -170,12 +170,27 @@ Inappropriate postings include:
 
 # Security Considerations
 
-The IETF Chair, the IETF Executive Director, or a sergeant-at-arms {{SAA-SOP}}
-appointed by the Chair is empowered to restrict posting by a person, or of a
-thread, when the content is inappropriate and represents a pattern of abuse
-{{SAA-UPC}}. They are encouraged to take into account the overall nature of the
-postings by an individual and whether particular postings are an aberration or
-typical. Complaints regarding their decisions should be referred to the IAB.
+A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to
+restrict posting by a person, or of a thread, when the content is
+inappropriate and represents a pattern of abuse. They are encouraged to
+take into account the overall nature of the postings by an individual
+and whether particular postings are an aberration or typical.
+
+The sergeants-at-arms are intended to establish a self-moderation
+function on the community, by the community. The IETF Chair therefore
+SHOULD NOT appoint an SAA who is serving in a NomCom-appointed IETF
+leadership position. If an SAA is selected for such a position, they
+SHOULD consider stepping down as SAA.
```

Comment:
```suggestion
SHOULD step down as SAA.
```

We don't need to go all RFC 6919 here.

Reply:
And imho we don't need to go all RFC 2119 either.

Reply:
```suggestion
will step down as SAA.
```